packages/llm-bot/test/constants.ts

```diff
@@ -0,0 +1,24 @@
+import { formatBytes32String, parseUnits } from "./utils";
```

Comment:
Are some/all of these test utilities files copied? If so, could you comment that in the PR so we know which to review as fresh code?